src/index.js

```diff
@@ -3,7 +3,8 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import { Provider } from 'react-redux';
 import App from './App';
-import store from './store/configureStore';
+// import store from './store/configureStore';
+import store from './store/entities';
```

Comment:
Modifying this code would break other team member's code, kindly stick to previously agreed code structure.